wrangler-service/src/main/java/io/cdap/wrangler/service/directive/DirectivesHandler.java

```diff
@@ -664,6 +664,10 @@ private String addLoadablePragmaDirectives(String namespace, Request request) {
       // Directives in this context).
       CompileStatus status = compiler.compile(new MigrateToV2(request.getRecipe().getDirectives()).migrate());
       RecipeSymbol symbols = status.getSymbols();
+      if (symbols == null) {
+        return null;
```

Comment:
Annotate the method with `@Nullable` since now it can return `null`. Also, have to check the caller of this method that it can handle `null`?

Reply:
+1

Reply:
Added Nullable annotation. Caller has a null check so it is safe